assets/sass/_form.scss

```diff
@@ -0,0 +1,3 @@
+.contact-form--customer-quote {
+    padding: 35px 40px 0;
+}
```

Comment:
Nit: newline at end of file. Also, would `.contact-form .customer-quote` or `form .customer-quote` be more appropriate selectors?